src/Routes/Devices/Inventory.js

```diff
@@ -265,7 +265,7 @@ const Inventory = ({
           selectedItems={setCheckedDeviceIds}
           selectedItemsUpdateable={canBeUpdated()}
           kebabItems={
-            groupActionsEnabled
+            !groupActionsEnabled
```

Comment:
I liked the latest changes too much , can you rename this variable to  hideGroupsActions and do the same changes as in DeviceTable 
and use the const FEATURE_HIDDEN_GROUP_ACTIONS

Reply:
changes applied, thanks